source/renderer/app/api/utils/patchAdaApi.js

```diff
@@ -48,7 +36,7 @@ export default (api: AdaApi) => {
           : get(sync_progress, 'quantity', 0);
 
       // extract relevant data before sending to NetworkStatusStore
-      const response = {
+      return {
         syncProgress: SYNC_PROGRESS || syncProgress,
```

Comment:
@disassembler this should be `syncProgress: syncProgress || SYNC_PROGRESS,`

Reply:
@tomothespian no it should not - we need to use `SYNC_PROGRESS` in case it was set as this is patched API - we modify the `SYNC_PROGRESS` variable in the scope of our tests.